File: c++/1-basic/1-grade-nilai.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
  int nilai = 80;

  if (nilai > 100 && nilai < 0) {
    cout << "Nilai invalid" << endl;
  } else if (nilai < 35) {
    cout << "E" << endl;
  } else if (nilai < 50) {
    cout << "D" << endl;
  } else if (nilai < 65) {
    cout << "C" << endl;
  } else if (nilai < 80) {
    cout << "B" << endl;
  } else if (nilai <= 100) {
    cout << "A" << endl;
  }
}

File: go/1-basic/1-grade-nilai.go
package main

import(
	"fmt"
)

func main() {
	var studentScore int = 80
	// var alphabet string = "SEPULSA OKE"
	// fmt.Scan(&studentScore)

	// Your Solution Code Here
	if studentScore < 0 || studentScore > 100 {
		fmt.Println("Nilai Invalid")
	} else if studentScore >= 80 {
		fmt.Println("Nilai A")
	} else if studentScore >= 65 {
		fmt.Println("Nilai B")
	} else if studentScore >= 50 {
		fmt.Println("Nilai C")
	} else if studentScore >= 35 {
		fmt.Println("Nilai D")
	} else if studentScore >= 0 {
		fmt.Println("Nilai E")
	}

	// Output
	// Nilai A

	fmt.Println(studentScore)
}